app/views/projects/tree_filter/show.html.haml

```diff
@@ -0,0 +1,24 @@
+- ref = @ref
+- project = @project
+- tree_filter_list = @tree_filter_list
+- path = @path
+- id = @id
+- commit = @commit
+
+- page_title path.presence || "Files", ref
```

Comment:
Since we don't actually take `@path` into account and always search from the root, I think we can just do `page_title "Find File", ref`